### ISSUE TITLE
[a11y] Fix accessibility violations in Create Tearsheet

### ISSUE THREAD

**Issue by anamikaanu96** [carbon-design-system/ibm-products#7562]
https://github.com/carbon-design-system/ibm-products/pull/7533#issuecomment

Once the violations are resolved, remove the skip and enable the accessibility test in CreateTearsheet.
